### DISCUSSION TITLE
What is the plan for documentation?

### DISCUSSION THREAD

**Discussion by conorfleming** [Open-SES/OpenSES#6] [Q&A]
Hi, what is the plan for OpenSES documentation?

- **Location** - is it to be hosted on the Wiki pages, under the 'OpenSES/manuals' folder in the repository, Google Docs, or somewhere else?
- **Format** - Github Markdown, LaTeX, or something else?
- **Contributions** - if this is hosted under 'OpenSES/manuals' then anyone could contribute (safely) with a pull request, but would need to understand how Git works. The Wiki would be easier to contribute to, particularly for non-programmers, but this would require opening access to anyone, and is riskier - there is no opportunity for review/approval before publication. Another alternative would be using Google Docs, with contributions suggested as comments, to be adopted/rejected by a project member who controls the document.

**Comment by TomODwyer-SES** [comment]
Hi Conor,
I am a little partial to Latex, similar to how FDS-SMV assemble their documentation, but am open to other, perhaps easier, methods.
Welcome,
-Tom
